SocialCareCaseViewerApi.Tests/V1/Helpers/TestHelpers.cs

```diff
@@ -634,5 +633,32 @@ public static GetTeamsRequest CreateGetTeamsRequest(string? contextFlag = null)
                 .RuleFor(q => q.PersonID, personID)
                 .RuleFor(q => q.PruneUnfinished, f => pruneUnfinished ?? f.Random.Bool());
         }
+
+        private static List<CaseStatusValue> CreateCaseStatusValues()
+        {
+            var caseStatusValues = new List<CaseStatusValue>();
+
+            for (var i = 0; i < new Random().Next(1, 10); i++)
+            {
+                var value = new Faker<CaseStatusValue>()
+                    .RuleFor(c => c.Name, f => f.Random.String2(1000))
```

Comment:
If these are entered into a separate table for answers, could we run the risk of the `name` or `value` being too large leading to flaky tests?